tests/e2e/mocha.opts

```diff
@@ -12,7 +12,8 @@
 
 ../packages/analytics/e2e/*.e2e.js
 
-../packages/auth/e2e/*.e2e.js
+# FIXME temporary, API limits make these failure-prone, toggled off during development
+#../packages/auth/e2e/*.e2e.js
```

Comment:
Doesn't need to be done now but just leaving a note as a reminder. We can also switch this to the auth emulator to avoid limits, but we can do that in a follow up PR that adds the auth emulator apis to RNFB as a feature I think.

Reply:
That is the very next feature on my personal list, I can hardly wait, sincerely